pkg/crc/machine/errors.go

```diff
@@ -0,0 +1,64 @@
+package machine
+
+import "github.com/code-ready/crc/pkg/crc/errors"
+
+func startError(name string, description string, err error) (StartResult, error) {
```

Comment:
Yup, very nice to have the error creation centralized somewhere as this was indeed not always fully consistent. One minor nit as always, I'm wondering if `reportStartError` or such would have been a better name.